benchmarks/README.md

```diff
@@ -0,0 +1,76 @@
+## Benchmarks
+
+The purpose of this tool is to benchmark the perf. of the 'Hermit-cli'.This will let us track performance changes from one release to another.
```

Comment:
```suggestion
The purpose of this tool is to benchmark the performance of the `hermit` CLI. This will let us track performance changes from one release to another.
```